Packages/Tracking/Physical Hands/Runtime/Scripts/Helpers/GrabHelper.cs

```diff
@@ -25,8 +25,8 @@ public class GrabHelper : MonoBehaviour
                 "\n\nWarning: Hard Contact hands can jitter when moving a kinematic object")]
         public bool useNonKinematicMovementOnly = true;
 
-        private ContactHand _leftContactHand { get { return physicalHandsManager.ContactParent.LeftHand; } }
```

Comment:
@MattFiler where did the physical hand code changes come from - was it you or was it left over from Bryce?

Reply:
This was changed in #1646